src/Views/index.js

```diff
@@ -15,7 +15,7 @@ export { Instructor } from "./Pages/Instructor";
 export { Profile } from "./Pages/Profile";
 
 export { Dictionary } from "./Tools/Dictionary";
-export { Flashcards } from "./Tools/Flashcards";
+export { Flashcards } from "./Tools/FlashCards";
```

Comment:
This should be Flashcards. I'm not sure how it keeps getting changed. Please revert back.